.githooks/pre-commit

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
You should look at https://typicode.github.io/husky/#/ - It is much much easier ;)

Reply:
I'm going to add that as an issue because I'd like to move on from hooks for now.

I've seen husky, and tried it once or twice, and honestly just now got hooks... How is husky better / easier than git hooks? Honest question here! Thank you @sarfata

Reply:
Spoke about this on our one-on-one, resolved